core/vpn/src/network.rs

```diff
@@ -400,7 +423,7 @@ impl Handler<Connect> for Vpn {
             Err(err) => return ActorResponse::reply(Err(err)),
         };
 
-        log::info!("VPN {}: connecting to {:?}", self.vpn.id(), remote);
+        log::info!("VPN {}: connecting to {remote:?}", self.vpn.id());
```

Comment:
I don't agree with this approach. As long as you can't evaluate functions in formatting expression, you will be always forced to use this ugly longer version in each case where you have at least one problematic expression

Reply:
You can alternatively move `self.vpn.id()` into separate variable if string interpolation is desired so much.
My problem with mixed style is that I need to scan _both_ the text and the parameter list to see what is logged. Otherwise when there is a parameter I know I only need to check the parameter list to see that.
```suggestion
        let vpn_id = self.vpn.id();
        log::info!("VPN {vpn_id}: connecting to {remote:?}");
```

Reply:
For me its not a problem, maybe I'm reading code differently.
Ok, I will commit this version, but to be honest, I don't like it is well.